metadata-ingestion/src/datahub/ingestion/source/usage/usage_common.py

```diff
@@ -77,7 +84,7 @@ def make_usage_workunit(
             uniqueUserCount=len(self.userFreq),
             totalSqlQueries=self.queryCount,
             topSqlQueries=[
-                self.trim_query(query, budget_per_query)
+                self.indent_query(self.trim_query(query, budget_per_query), indent)
```

Comment:
The order should be the other way round... trim(indent)...otherwise, indent might add more characters, defeating what trim has tried to do which is to limit the max size of the query string.

Reply:
ok I permuted

Reply:
Thank you Corentin!